### PR TITLE
Inline scripts formatting

### PR DESCRIPTION
**Pull request by PolinaShneider**
Formatting JavaScript content of inline (`script =`) scripts

inline_formatting

### CODE REVIEW

File: server/src/test/scriptFormatting.test.ts [hunk 1]
```diff
@@ -95,4 +95,45 @@ endscript`;
...
+
+    test("Needs formatting. Multiple spaces before and after equals sign", () => {
+        const text = `[configuration]
+  script            =       var hello= value()`;
+        const options: FormattingOptions = FormattingOptions.create(2, true);
+        const expected: TextEdit[] = [
+            TextEdit.replace(Range.create(
+                Position.create(1, 22),
+                Position.create(1, 46)),
+                "var hello = value()"
```
Comment (annstriganova) [inline server/src/test/scriptFormatting.test.ts]: On `master`:
screenscast_20190712_151309

On `javascript-inline-formatting`:
screenscast_20190712_151549

Comment (PolinaShneider) [reply server/src/test/scriptFormatting.test.ts]: Fixed it

**Comment by PolinaShneider** [comment]
Faced several difficulties when transferring this PR to `charts-ls`. Now we have special language formatter [factory](https://github.com/axibase/charts-language-service/blob/master/src/nestedCodeFormatter.ts#L13), which returns specific formatter instance based on language id. At the moment of this PR creation that logic wasn’t implemented yet.

I think, it could be useful to expand language formatter API to `block` and `inline` formatting, rather than adding inline js formatter as a new instance, because no matter block or inline — js remains js :)

Will do some refactoring 

**P.S.** Inline and block js formatting is quite different and cannot be processed in one method

**Comment by PolinaShneider** [comment]
Closing PR, because it was transferred to axibase/charts-language-service#70

**Pull request closed without merging**